apps/ejabberd/include/ejabberd.hrl

```diff
@@ -70,6 +70,9 @@
 
 -type scram() :: #scram{}.
 
--record(route, {domain, handler}).
+-record(route, {
+          domain :: binary(),
+          handler :: mongoose_packet_handler:t()
```

Comment:
We've agreed in previous Elvis PR that we want to use `t()` as a main type for mongoose behaviours.